specification/sql/resource-manager/readme.md

```diff
@@ -998,6 +997,7 @@ input-file:
 - ./Microsoft.Sql/preview/2022-11-01-preview/DatabaseSqlVulnerabilityAssessmentScanResult.json
 - ./Microsoft.Sql/preview/2022-11-01-preview/DatabaseSqlVulnerabilityAssessmentScans.json
 - ./Microsoft.Sql/preview/2022-11-01-preview/DatabaseSqlVulnerabilityAssessmentsSettings.json
+- ./Microsoft.Sql/preview/2023-02-01-preview/FailoverGroups.json
```

Comment:
we have not release 2023-02-01-preview. please hold this PR until late next week

Reply:
Any updates on when the preview will be merged in?